### DISCUSSION TITLE
Adding search to Virtuoso docs?

### DISCUSSION THREAD

**Discussion by mkosir** [petyosi/react-virtuoso#960] [Ideas]
Hi 👋 , thanks for great library!
Just wondering, since Virtuoso is built with Docusaurus, which makes it trivial to include search functionality through the docs, is there any hold back to have search?
I think it would be pretty useful addition for devs to quickly find info.

**Comment by petyosi** [comment]
No holdback apart from my own time being in deficit. Happy to accept a PR.
